go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/giantswarm/helmclient v0.2.0
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient v0.2.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.1.2
+	github.com/giantswarm/k8scloudconfig/v6 v6.1.3-0.20200827212125-e3aa902e48b2
```

Comment:
Will update to v6.1.3 when I merge https://github.com/giantswarm/k8scloudconfig/pull/784 tomorrow.